net-mgmt/pfSense-pkg-softflowd/files/usr/local/pkg/softflowd.inc

```diff
@@ -56,6 +56,9 @@ function sync_package_softflowd() {
 			if (is_numericint($cf['hoplimit'])) {
 				$start .= " -L " . escapeshellarg($cf['hoplimit']);
 			}
+			if (is_numericint($cf['sample']) && $cf['sample'] > 0) {
```

Comment:
`$cf['sample'] > 0` should be surrounded by parenthesis.

Reply:
fixed